.github/CONTRIBUTING.md

```diff
@@ -1,29 +1,5 @@
-**The Azure IoT SDKs team wants to hear from you!**
+Hi there!
 
-- [Ask a question](#ask-a-question)
-- [File a bug](#file-a-bug)
-- [Contribute documentation](#contribute-documentation)
-- [Contribute code](#contribute-code)
+**iothub-explorer** will be retired on November 31st, 2018 and has been replaced by the [Azure CLI IoT Extension](https://aka.ms/iotcli).
 
-# Ask a question
-Our team monitors Stack Overflow, especially the [azure-iot-hub](http://stackoverflow.com/questions/tagged/azure-iot-hub) tag. It really is the best place to ask.
-
-We monitor the Github issues section specifically for bugs found with our SDK and tools, however we will reply to questions asked using Github issues too.
-
-# File a bug (code or documentation)
-That is definitely something we want to hear about. Please open an issue on github, we'll address it as fast as possible. Typically here's the information we're going to ask for to get started:
-- What version of `iothub-explorer` are you using?
-- Do you have logs showing what's happening?
-- Do you have code samples of what is/should be happening on the other side?
-
-`iothub-explorer` is entirely open-source and we do accept pull-requests if you feel like taking a stab at fixing the bug and maybe adding your name to our commit history :) Please mention any relevant issue number in the pull request description.
-
-# Contribute code and/or documentation
-We look at all pull requests submitted against the `master` branch carefully. We also actively use the [Wiki](https://github.com/Azure/azure-iot-sdk-node/wiki) for longer-form documents. The wiki can be cloned and used as a regular Git repository so feel free to contribute there too!
-
-As far as code is concerned the code associated with the PR will be pulled and run through the gated build before it makes it to the master branch. As much as possible, we insist on having tests associated with the code, and if necessary, additions/modifications to the requirement documents. As a matter of fact, the build will fail if code coverage goes down.
-
-Also, have you signed the [Contribution License Agreement](https://cla.microsoft.com/) ([CLA](https://cla.microsoft.com/))? A friendly bot will remind you about it when you submit your pull-request.
-
-If you feel like your contribution is going to be a major effort, you should probably give us a heads-up. We have a lot of items captured in our backlog and we release every two weeks, so before you spend the time, just check with us to make
-sure your plans and ours are in sync :) Just open an issue on github and tag it "enhancement" or "feature request"
+We will not address any issue regarding iothub-explorer except those related to the migration to the azure cli.
```

Comment:
Capitalize Azure CLI

Reply:
consider replacing text with 'We will not address any issues regarding iothub-explorer except for those related to migrating towards the Azure CLI.'